uefi-raw/src/protocol/device_path.rs

```diff
@@ -1,4 +1,60 @@
-use crate::{guid, Char16, Guid};
+/*
```

Comment:
I don't think that this is necessary. Right, @nicholasbishop ?

Reply:
I agree; we're just adding types defined in the UEFI spec here, no need to add the EDK2 license.